src/azul/__init__.py

```diff
@@ -27,6 +27,10 @@ def es_endpoint(self) -> Tuple[str, int]:
             host, _, port = es_endpoint.partition(':')
             return host, int(port)
 
+    @property
+    def home_directory(self) -> str:
```

Comment:
Could you rename this to `project_root`?

My apologies, I should have specififed that earlier.